calibration/CTDMOG/CGINS-CTDMOG-13434__20161020.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13434,CC_ptcb1,-3.000000e-004,
 37-13434,CC_ptcb2,0.00E+00,
 37-13434,CC_ptempa0,1.486324e+002,
-37-13434,CC_ptempa1,-6.712965e-002,
-37-13434,CC_ptempa2,5.804383e-007,
-37-13434,CC_wbotc,2.7651e-007,
+37-13434,CC_ptempa1,-6.712964e-02,
+37-13434,CC_ptempa2,5.804384e-07,
+37-13434,CC_wbotc,2.765135e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT